### PR TITLE
layout_landing-page

### PR DESCRIPTION
**Pull request by iLinuxx**
[DEMO](https://ilinuxx.github.io/layout_landing-page/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,454 @@
...
+        <div class="header__content">
+          <div class="top-bar">
+            <a
+              href="#"
+              alt="THE MET logo"
+              class="top-bar__logo"
+            >
+            </a>
+            <div class="top-bar__icons">
+              <a href="tel: +1 212-535-7710" class="icon icon--phone">
```
Comment (volodymyr-soltys97) [inline src/index.html:30]: Need to fix format phone only in href everywhere
```suggestion
              <a href="tel:12125357710" class="icon icon--phone">
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good start 👍 
Let's improve your page
1. Add `cursor: pointer` here
image

2. Make these fields is required before clicking send

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,457 @@
...
+            </ul>
+          </nav>
+          <div class="numbers">
+            <a href="tel:12125357710" class="numbers__phone">
+              +1 212-535-7710
+            </a>
+            <p class="numbers__phone--title">
+              The Met Fifth Avenue
+            </p>
+            <a href="tel:12129233700" class="numbers__phone">
```
Comment (volodymyr-soltys97) [inline src/index.html:106]: Need to add it everywhere
```suggestion
            <a href="tel:+12129233700" class="numbers__phone">
```

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,457 @@
...
+              >
+              <textarea
+              class="contacts__form__input contacts__form__input--textarea"
+              name="Message"
+              id="contacts__form__input"
+              cols="30"
+              rows="10"
+              placeholder="Message"
+              required
+              ></textarea>
```
Comment (volodymyr-soltys97) [inline src/index.html:440]:
```suggestion
              <textarea
                class="contacts__form__input contacts__form__input--textarea"
                name="Message"
                id="contacts__form__input"
                cols="30"
                rows="10"
                placeholder="Message"
                required
              ></textarea>
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Images ovelapping menu. To fix increase z-index for menu
2023-08-03_15h50_19

Check this case
https://github.com/mate-academy/layout_landing-page/assets/61904995/29a6c33c-2916-4682-9800-40db3e6d8285

**Pull request closed without merging**